script/hardhat/fork/govUpdateTest.ts

```diff
@@ -0,0 +1,326 @@
+        await mineUpTo((await ethers.provider.getBlockNumber()) + PROPOSAL_TYPE_CONFIGS[0].votingPeriod);
+        const tx = await governorProxy.connect(proposer).queue(proposalId);
+        await tx.wait();
+      });
+    });
+    describe("Execute Proposal:", () => {
+      it("should revert if timelock period has not passed", async () => {
+        await expect(governorProxy.connect(proposer).execute(proposalId)).to.be.revertedWith(
+          "Timelock::executeTransaction: Transaction hasn't surpassed time lock.",
+        );
+      });
+      it("should be executed successfully", async () => {
+        await mineUpTo((await ethers.provider.getBlockNumber()) + TIMELOCK_DELAYS_MAINNET.NORMAL);
+        const tx = await governorProxy.connect(proposer).execute(proposalId);
+        await tx.wait();
+      });
+      it("correct value should be set after execution", async () => {
+        const liquidationIncentive = await comptrollerProxy.liquidationIncentiveMantissa();
+        expect(liquidationIncentive).to.equal(NEW_LIQ_INCENTIVE);
+      });
+    });
+  });
+  describe("FASTTRACK VIP", () => {
+    const proposal = {
+      signatures: ["_setLiquidationIncentive(uint256)"],
+      targets: [COMPTROLLER_PROXY_MAINNET],
+      values: ["0"],
+      params: [[NEW_LIQ_INCENTIVE]],
+    };
+    let proposalId;
+    before(async () => {
+      await loadFixture(governanceFixture);
+    });
+    describe("Propose:", () => {
+      it("should revert if below votingThreshold", async () => {
+        const { targets, signatures, values } = proposal;
+        await expect(
+          governorProxy.propose(targets, values, signatures, getCalldatas(proposal), "VIP 2", PROPOSAL_TYPES.FASTTRACK),
+        ).to.be.revertedWith("GovernorBravo::propose: proposer votes below proposal threshold");
+      });
+      it("should pass", async () => {
+        const { targets, signatures, values } = proposal;
+        const proposalIdBefore = await governorProxy.callStatic.proposalCount();
+        const tx = await governorProxy
+          .connect(proposer)
+          .propose(targets, values, signatures, getCalldatas(proposal), "VIP 2", PROPOSAL_TYPES.FASTTRACK);
+        await tx.wait();
+        proposalId = await governorProxy.callStatic.proposalCount();
+        expect(proposalIdBefore.add(1)).to.equal(proposalId);
+      });
+    });
+    describe("Cast Vote:", () => {
+      it("should cast votes", async () => {
+        await mine();
+        await expect(governorProxy.connect(proposer).castVote(proposalId, 1))
+          .to.emit(governorProxy, "VoteCast")
+          .withArgs(await proposer.getAddress(), proposalId, 1, 335338539326609831760663n, "");
+
+        await expect(governorProxy.connect(supporter).castVote(proposalId, 1))
+          .to.emit(governorProxy, "VoteCast")
+          .withArgs(await supporter.getAddress(), proposalId, 1, 846441499442385723656125n, "");
+      });
+    });
+    describe("Queue Proposal:", () => {
```

Comment:
```suggestion
    describe("Queue Proposal", () => {
```
I am fairly certain that hardhat will add necessary punctuation when rendering test output